src/app/archbishop/archbishop.py

```diff
@@ -57,6 +71,10 @@ def handle_waffle(message, say):
         say(response)
         return Response(messages.load("event.request.handled"), status=200)
 
+    def get_avg_score(player):
+        # Return average score to 2 decimal places
+        return round(player.score / player.games, 2)
+
```

Comment:
How about putting this on the player itself?

`player.average()` -> `return round(self.score / self.games, 2)`

Reply:
That makes infinitely more sense. Im clearly not on the ball this morning!

Reply:
Updated now. Please Review